conf/radiusd/packetfence-tunnel.example

```diff
@@ -20,13 +20,14 @@ server packetfence-tunnel {
 #  Make *sure* that 'preprocess' comes before any realm if you
 #  need to setup hints for the remote radius server
 authorize {
-    # TTLS does not send an EAP-Message to be parsed so the eap module
-    # cannot assign the EAP-Type
-    if ( outer.EAP-Type == TTLS) {
-        update request {
-            &EAP-Type := TTLS
-        }
+  # TTLS does not send an EAP-Message to be parsed so the eap module
```

Comment:
Just fix the indentation here so it matches the rest of the authorize section and I'll merge

Reply:
Fixaide